src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -78,6 +88,11 @@ public RawScriptResult(int? statusCode, object content)
                 selectedFormatter = SelectedFormatter(formatterContext);
             }
 
+            if (!string.IsNullOrEmpty(contentType))
+            {
+                formatterContext.ContentType = contentType;
```

Comment:
Remove this (and local variable) and set the content type when the context is created (in `CreateFormatterContext`) like the following (no need for a local variable or `null` check):
``` csharp
        private static OutputFormatterWriteContext CreateFormatterContext(HttpResponse response, object content)
        {
            var context = new OutputFormatterWriteContext(
                response.HttpContext,
                (s, e) => new HttpResponseStreamWriter(s, e),
                content?.GetType(),
                content);

            context.ContentType = response.ContentType;

            return context;
        }
```